src/Array2D.luna

```diff
@@ -0,0 +1,235 @@
+import Std.Foreign.C.Value
+import Std.Foreign
+
+import Dataframes.Utils
+import Dataframes.Types
+
+class Size:
+    SizeVal:
+        rows   :: Int
+        columns:: Int
+
+    def getRows: case self of
```

Comment:
No need for getters if you have named fields. But you need at least one method, so I recommend `toText`.

Reply:
Also, `ShapeVal` seems kind of internal (because it is, right? That's our workaround for Luna not being able to declare methods on class level, which will be changed in the future to something really expressive) so I would think about creating a "factory" method like `Array2D.make`, maybe `new`?

It's up to you to decide whether `Size.new 1 2` looks better or worse than `SizeVal 1 2` :P